node_name: 'thrust_allocation'

MAPP:
  MAPP1: [-2.565,-2.4594,-2.3218,-2.1647,-2.0261,-1.8925,-1.7672,-1.6417,-1.5295,-1.3971,-1.2786,-1.1559,-1.0347,-0.93514,-0.84887,-0.76416,-0.67639,-0.5881,-0.51947,-0.4709,-0.41047,-0.34581,-0.27869,-0.22347,-0.18142,-0.14023,-0.10666,-0.08283,-0.053513,-0.03441,-0.00075815,0.00075815,0.03441,0.053513,0.08283,0.10666,0.14023,0.18142,0.22347,0.27869,0.34581,0.41047,0.4709,0.51947,0.5881,0.67639,0.76416,0.84887,0.93514,1.0347,1.1559,1.2786,1.3971,1.5295,1.6417,1.7672,1.8925,2.0261,2.1647,2.3218,2.4594,2.565]
  MAPP2: [-2.2109,-2.1357,-1.9632,-1.8187,-1.6812,-1.5452,-1.4137,-1.3073,-1.17,-1.0763,-0.9812,-0.88022,-0.77893,-0.68714,-0.61917,-0.53949,-0.45877,-0.3901,-0.32101,-0.27918,-0.24677,-0.1989,-0.16348,-0.13117,-0.098309,-0.083152,0.083152,0.098309,0.13117,0.16348,0.1989,0.24677,0.27918,0.32101,0.3901,0.45877,0.53949,0.61917,0.68714,0.77893,0.88022,0.9812,1.0763,1.17,1.3073,1.4137,1.5452,1.6812,1.8187,1.9632,2.1357,2.2109
  ]
  MAPP3: [2.565,2.4594,2.3218,2.1647,2.0261,1.8925,1.7672,1.6417,1.5295,1.3971,1.2786,1.1559,1.0347,0.93514,0.84887,0.76416,0.67639,0.5881,0.51947,0.4709,0.41047,0.34581,0.27869,0.22347,0.18142,0.14023,0.10666,0.08283,0.053513,0.03441,0.00075815,-0.00075815,-0.03441,-0.053513,-0.08283,-0.10666,-0.14023,-0.18142,-0.22347,-0.27869,-0.34581,-0.41047,-0.4709,-0.51947,-0.5881,-0.67639,-0.76416,-0.84887,-0.93514,-1.0347,-1.1559,-1.2786,-1.3971,-1.5295,-1.6417,-1.7672,-1.8925,-2.0261,-2.1647,-2.3218,-2.4594,-2.565]

THRUST:
  Th1: [43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118]
  Th2: [46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114]
  Th3: [43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118]
